.changeset/social-cougars-push.md

```diff
@@ -0,0 +1,6 @@
+---
+"@digdir/designsystemet-theme": patch
+"@digdir/designsystemet": patch
+---
+
+Remove fragmented theme CSS files from `tokens build`.
```

Comment:
```suggestion
Remove individual theme color CSS files from `tokens build`.
```
Not sure about the wording here, but "fragmented" seems a bit non-decriptive to me